qdl.c

```diff
@@ -55,6 +55,7 @@ enum {
 };
 
 bool qdl_debug;
+bool allow_missing;
```

Comment:
This is only used as a local variable inside main(), so please move it there (and initialize it to 0).

Reply:
fixed.